psp/models/recent_history.py

```diff
@@ -204,21 +216,32 @@ def set_data_sources(
         *,
         pv_data_source: PvDataSource,
         nwp_data_sources: dict[str, NwpDataSource] | None = None,
+        sat_data_sources: dict[str, SatelliteDataSource] | None = None,
     ):
         """Set the data sources.
 
         This has to be called after deserializing a model using `load_model`.
         """
         self._pv_data_source = pv_data_source
         self._nwp_data_sources = nwp_data_sources
+        self._satellite_data_sources = sat_data_sources
```

Comment:
self._satellite_data_sources = sat_data_sources, similar naming convention to use satellite and sat instead of just sat

Reply:
Ive got for `satellite_data_sources`, as it make its clear